sdk/spring/azure-spring-boot-test-aad-resource-server/src/test/java/com/azure/test/aad/resource/server/it/AADWeiResourceServerIT.java

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.test.aad.resource.server;
+package com.azure.test.aad.resource.server.it;
```

Comment:
why are we adding `it` to the package?

Reply:
Sorry, I haven’t finished it yet, I’m for testing and will change it later.

Reply:
No worries, if this PR is not ready for reviewing, you could change it to a draft PR.